saylua/modules/forums/templates/main.html

```diff
@@ -4,7 +4,19 @@
   <link rel="stylesheet" type="text/css" href="{{ include_static('css/forums.min.css') }}">
 {% endblock %}
 {% block body %}
-<h1>Forums</h1>
+<h1>
+  <i class="fa fa-fw fa-comment" aria-hidden="true"></i>
```

Comment:
Good call. I wasn't sure if this would be necessary (or desirable) here since the icon is more decorative than functional? Though there are at least a dozen places where I've been lazy about adding title/alt text where it could make sense.